rustfmt.toml

```diff
@@ -0,0 +1,10 @@
+# stable
+newline_style = "Unix"
+hard_tabs = false
+tab_spaces = 4
+reorder_imports = true
+reorder_modules = true
+
+# These are nightly
+# group_imports = "StdExternalCrate"
+# imports_granularity = "Crate"
```

Comment:
Do you have to uncomment them to format?

Reply:
Yes, because they are `nightly` only